docs/guides/run-jobs-session.mdx

```diff
@@ -65,15 +65,19 @@ The context manager automatically opens and closes the session.
 <Tabs>
   <TabItem value="PrimV2" label="V2 primitives">
 ``` python
+from qiskit_ibm_runtime import Session, Samplerv2 as Sampler, Estimatorv2 as Estimator
```

Comment:
The original config I was using didn't like `estimatorv2`. I added a generic rule so that any word with a number is fine, since it's probably code. So, we can go back to `estimatorv2` if we want. 

But I figured it's better to switch to the same style we use elsewhere.